### PR TITLE
completed html-syntax

### PR DESCRIPTION
**Pull request by lucachitayat**
image

### CODE REVIEW

File: html-syntax/index.html [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+  </head>
+  <body>
+    <img
+      title="wabuffet"
+      width="500"
+      src="https://www.serebii.net/swordshield/pokemon/202.png"
+      alt="Get Wabuffeted, Bro" />
+    <h1>Wabuffet</h1>
+    <p>You both know and love Him</p>
+    <p>©Pokemon ®Nintendo</p>
```
Comment (robgmerrill) [inline html-syntax/index.html:16]: We want to use HTML entities for these: https://developer.mozilla.org/en-US/docs/Learn/HTML/Introduction_to_HTML/Getting_started#entity_references_including_special_characters_in_html

**Review by robgmerrill — CHANGES_REQUESTED** [review]
Left some feedback on using HTML entities.

File: html-syntax/index.html (hunk 1 above)
Comment (lucachitayat) [reply html-syntax/index.html]: entities updated in quiz questions push

**Review by robgmerrill — APPROVED** [review]
👍

**Pull request merged**